tests/k8s_ingress_test_new/diff_output_from_k8s_ingress_test.csv

```diff
@@ -0,0 +1,47 @@
+source,destination,dir1,dir2,diff-type
+default/reviews-v1-545db77b95[ReplicaSet],default/productpage-v1-6b746f74dc[ReplicaSet],All Connections,TCP 9080,changed
+default/reviews-v1-545db77b95[ReplicaSet],default/ratings-v1-b6994bb9[ReplicaSet],All Connections,TCP 9080,changed
+default/reviews-v2-7bf8c9648f[ReplicaSet],default/productpage-v1-6b746f74dc[ReplicaSet],All Connections,TCP 9080,changed
+default/reviews-v2-7bf8c9648f[ReplicaSet],default/ratings-v1-b6994bb9[ReplicaSet],All Connections,TCP 9080,changed
+default/reviews-v3-84779c7bbc[ReplicaSet],default/productpage-v1-6b746f74dc[ReplicaSet],All Connections,TCP 9080,changed
+default/reviews-v3-84779c7bbc[ReplicaSet],default/ratings-v1-b6994bb9[ReplicaSet],All Connections,TCP 9080,changed
+0.0.0.0-255.255.255.255,default/unicorn[Deployment],No Connections,All Connections,added
+default/unicorn[Deployment],0.0.0.0-255.255.255.255,No Connections,All Connections,added
+default/unicorn[Deployment],default/details-v1-79f774bdb9[ReplicaSet],No Connections,TCP 9080,added
+{ingress-controller},default/unicorn[Deployment],No Connections,TCP 8080,added
```

Comment:
let's place differences with `{ingress-controller}` at the end, similar to connlist

Reply:
done